charts/ia-home-office-integration-api/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: Helm chart for ia-home-office-integration-api microservice
 name: ia-home-office-integration-api
 home: https://github.com/hmcts/ia-home-office-integration-api
-version: 0.0.8
+version: 0.0.7
```

Comment:
We should not decrease version of Charts. As far as I know PE Team doesn't recommend that. Please add new version `0.0.9`

Reply:
upgraded to 0.0.9